charts/celeborn/values.yaml

```diff
@@ -36,10 +36,33 @@ workerReplicas: 5
 securityContext:
   fsGroup: 10006
 
+# Current Celeborn support followings volume type:
+# - emptyDir
+# - hostPath
+# Note: size only works in emptyDir type
+# Celeborn Master will pick first volumes for store raft log
+volumes:
+  master:
+    - path: /mnt/rss_ratis
```

Comment:
A potential improvement(we can do it in another PR if the current implementation meets the requirements): separate `mountPath` and `hostPath`

Reply:
+1 separate `mountPath ` & `hostPath ` for more flexibility, i will do it in another PR.